.gitignore

```diff
@@ -121,3 +121,4 @@ venv.bak/
 
 # NWB
 .nwb
+.pre-commit-config.yaml
```

Comment:
Were you able to get this setup separately to replace the main workflow?

Reply:
It is working locally for me.